File: problems/b/bijele/bijele.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>

using namespace std;

int main() {
    int k, q, r, b, n, p;

    cin >> k;
    cin >> q;
    cin >> r;
    cin >> b;
    cin >> n;
    cin >> p;

    // One king
    k = 1 - k;

    // One queen
    q = 1 - q;

    // Two rooks
    r = 2 - r;

    // Two bishops
    b = 2 - b;

    // Two knights
    n = 2 - n;

    // Eight pawns
    p = 8 - p;

    cout << k << " " << q << " " << r << " " << b << " " << n << " " << p << endl;
}

File: problems/b/bijele/bijele.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    correct_set = [1, 1, 2, 2, 2, 8]
    pieces = [correct_set[i] - int(p) for i, p in enumerate(input().split())]
    print(*pieces)

if __name__ == '__main__':
    main()